infrastructure/tests/baloon-pods.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: placeholder
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      pod: placeholder-pod
+  template:
+    metadata:
+      labels:
+        pod: placeholder-pod
+    spec:
+      priorityClassName: placeholder-priority
+      terminationGracePeriodSeconds: 0
+      containers:
+      - name: ubuntu
```

Comment:
Can we use a smaller image for this? I don't recall but `ubuntu` is a fairly bloated image for what we need to do here

Reply:
Switched to busybox

Reply:
busybox or alpine will be also smaller in size